sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuShuffledSizedHashJoinExec.scala

```diff
@@ -1138,14 +1138,17 @@ class KudoSpillableHostConcatResult(kudoTableHeader: KudoTableHeader,
 ) extends SpillableHostConcatResult  {
   require(kudoTableHeader != null, "KudoTableHeader cannot be null")
   require(hmb != null, "HostMemoryBuffer cannot be null")
+  val bufferLength: Long = hmb.getLength
 
-  override def toBatch: ColumnarBatch = closeOnExcept(buffer.getHostBuffer()) { hostBuf =>
-    KudoSerializedTableColumn.from(new KudoTable(kudoTableHeader, hostBuf))
+  override def toBatch: ColumnarBatch = closeOnExcept(spillableBuffer.getHostBuffer()) { hostBuf =>
+    KudoSerializedTableColumn.from(SpillableKudoTable(kudoTableHeader, hostBuf))
```

Comment:
nit, the `SerializedTableColumn.from` method takes a header and a buff, but the `KudoSerializedTableColumn.from` method takes a `SpillableKudoTable`. It would be cleaner if it took a header and a buff, but that's a nit.

Reply:
changed accordingly